src/datepicker/utils/bs-calendar-utils.ts

```diff
@@ -54,6 +54,14 @@ export function isDisabledDate(date: Date, datesDisabled: Date[]): boolean {
   return datesDisabled.some((dateDisabled: Date) => isSame(date, dateDisabled, 'date'));
 }
 
+export function isEnabledDate(date: Date, datesEnabled: Date[]): boolean {
+  if (datesEnabled === undefined || !datesEnabled || !datesEnabled.length) {
```

Comment:
What's the difference between `datesEnabled === undefined` and `!datesEnabled` ?
Where is check that datesEnabled isArray?
Where is check that array consists only from Date instance? What if I pass only numbers or strings to this array?

Reply:
if you pass something else, which is not a Date instance, that will not successfully pass isSame check, so, that nothing will be enabled / or disabled, because we have exactly the same function for datesDisabled 

About typecheck -> will be reworked right now